CNAME

```diff
@@ -0,0 +1 @@
+ydpatel0309.github
```

Comment:
where are you using CNAME in your codebase?

Reply:
CNAME came because I tried to change the github deployed page domain. After all, it was giving me a DNS error.